packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_controller.dart

```diff
@@ -144,8 +144,8 @@ class GoogleMapController {
     // Create the map...
     _googleMap = _createMap(_div, options);
 
-    _attachMapEvents(_googleMap);
-    _attachGeometryControllers(_googleMap);
+    _attachMapEvents(_googleMap!);
+    _attachGeometryControllers(_googleMap!);
```

Comment:
Optional: You could make a local, set `_googleMap` to the local, then use the local everywhere instead of `!`.

Reply:
Indeed, done.